tools/building.py

```diff
@@ -656,7 +658,8 @@ def move_to_safe_7bit_ascii_filename(filename):
   # 7-bit ASCII range. Therefore make sure the command line we pass does not contain any such
   # input files by passing all input filenames relative to the cwd. (user temp directory might
   # be in user's home directory, and user's profile name might contain unicode characters)
-  proc = run_process(cmd, stderr=PIPE, check=False, env=env, cwd=tempfiles.tmpdir)
+  # https://github.com/google/closure-compiler/issues/4159: Closure outputs stdout/stderr in iso-8859-1 on Windows.
+  proc = run_process(cmd, stderr=PIPE, check=False, env=env, cwd=tempfiles.tmpdir, encoding='iso-8859-1' if WINDOWS else 'utf-8')
```

Comment:
Is this and existing bug, or does it only show up with `--charset=UTF8` is added above?   (what is the default charset used by closure?

Reply:
This is an Closure compiler bug. It occurs both with and without `--charset=UTF8`, so does not relate to that.

I don't know what is the default charset used by Closure.